### ISSUE TITLE
Piloting Problems

### ISSUE THREAD

**Issue by zlwilson** [9khine/HelpKiosk#11]
Here are the new issues to fix that we found in the first pilot at GSC:

1. Green highlighting for feedback when steps are complete
2. Make all buttons look the same
3. Increase text size on the home page
4. Add a "Click Home Button" instruction

Possible changes, but would mean changing the text from exactly copying the manual which might be a problem:
5. Resolve "App Screen" and "Home Screen" confusion
6. Split compound instructions in Messages

**Comment by zlwilson** [comment]
Fixed 1.

The problem is we don't check every single step–some steps are marked done once the parent step is done because we have no way of checking (example: entering text in a Message)–but this mostly solves it!

**Comment by zlwilson** [comment]
Fixed 2 and 4.

All the buttons have the same gradient and border now. We could go for a different look but this works for now to make all the clickable elements look the same.
Added more text to the button at the bottom of the instruction, I think it looks ok with more text

**Comment by zlwilson** [comment]
Other possible fix:
7. Automatically highlight the first step so it's apparent it can be clicked

**Comment by zlwilson** [comment]
Opened new issue with latest pilot problems
